tests/OpenCloud/Tests/Database/DatabaseTestCase.php

```diff
@@ -37,4 +55,9 @@ public function setupObjects()
         $this->datastore = $this->service->datastore('10000000-0000-0000-0000-000000000001');
         $this->datastoreVersion = $this->datastore->version('b00000b0-00b0-0b00-00b0-000b000000bb');
     }
+
+    protected function assertCriticalMessageWasLogged()
+    {
+        $this->assertNotEmpty($this->getClient()->getLogger()->getCriticalLogMessage());
```

Comment:
yeah, this will do. another idea would be to use output buffering and actually check the stdout content [`ob-start`](http://php.net/manual/en/function.ob-start.php) and [`ob-get-contents`](http://php.net/manual/en/function.ob-get-contents.php)